docs/sources/reference/commandline/cli.md

```diff
@@ -98,16 +102,16 @@ you can write `docker run -it --name test busybox sh`.
 
 ### Boolean
 
-Boolean options take the form `-d=false`. The value you see in the help text is the
-default value which is set if you do **not** specify that flag. If you specify
-a Boolean flag without a value, this will set the flag to `true`, irrespective
-of the default value.
+Boolean options take the form `-d=false`. The value you see in the help text is
+the default value which is set if you do **not** specify that flag. If you
+specify a Boolean flag without a value, this will set the flag to `true`,
+irrespective of the default value.
 
-For example, running `docker run -d` will set the value to `true`, so
-your container **will** run in "detached" mode, in the background.
+For example, running `docker run -d` will set the value to `true`, so your
+container **will** run in "detached" mode, in the background.
 
-Options which default to `true` (e.g., `docker build --rm=true`) can only
-be set to the non-default value by explicitly setting them to `false`:
+Options which default to `true` (e.g., `docker build --rm=true`) can only be
```

Comment:
which > that 
inconsistent use of flag or option (not just use but maybe we can clean some up) -->

When a flag defaults to `true`, you can only set it to `false` by explicitly passing the flag: